plugins/modules/cloud/docker/docker_swarm_service.py

```diff
@@ -2434,6 +2448,8 @@ def get_service(self, name):
                 ds.networks.append(network)
         ds.service_version = raw_data['Version']['Index']
         ds.service_id = raw_data['ID']
+
+        ds.init = task_template_data['ContainerSpec'].get('Init')
```

Comment:
I'm not sure whether this makes sense, but maybe:
```suggestion
        ds.init = task_template_data['ContainerSpec'].get('Init', False)
```

Reply:
Yes, I agree your suggestion makes more sense.